### DISCUSSION TITLE
Feature request: add an option to focus on global volume by default

### DISCUSSION THREAD

**Discussion by aybe** [File-New-Project/EarTrumpet#1416] [Ideas]
In Windows 10, as soon as you'd open volume, you could immediately use mouse wheel to set the global volume.

Right now, there is an extra step: you have to focus global volume, then only you can change its level with mouse wheel.

**Comment by riverar** [comment]
@aybe I think you may have missed the mouse settings for this!

![image](https://github.com/File-New-Project/EarTrumpet/assets/475132/155d97d9-2c04-4a3f-9f76-f12bdc42aea1)

**Comment by aybe** [comment]
Not sure what's wrong but I don't have any of these settings:

<img width="578" alt="image" src="https://github.com/File-New-Project/EarTrumpet/assets/1537591/b138e6c1-c31a-4cdd-bb93-1827fdb83858">

**Reply by riverar** [reply]
Ah, these settings are currently in our dev branch (Click the link at the bottom of [eartrumpet.app](https://eartrumpet.app/).) I'll push out an update to production soon

**Comment by aybe** [comment]
Works great, thank you!
